### ISSUE TITLE
Sites - `get_energy_details` isn't calling the "correct" API

### ISSUE THREAD

**Issue by chrisjohnson00** [Yenthe666/pysolaredge#1]
SolarEdge has an API called `Site Energy -Detailed` with an endpoint of `/site/{siteId}/energyDetails`
This library includes a method called `get_energy_details`, but it doesn't call this API, it calls the Site Energy API (`/site/{siteId}/energy`).
I've corrected this in my project by overriding the `Sites` object and providing access to this endpoint.

I can PR this change to you, but I'd like you to provide me with direction on how to proceed?  To correct the API signature, would mean to create a breaking change to this project (which isn't 1.x, so do you care?).
In my override class I call it `get_full_energy_details` to add the functionality.  

Let me know how you'd want this incorporated into this project and I'll do so.

**Comment by Yenthe666** [comment]
Hm you're right, good catch! You can make a PR to fix this 👍
Comes back from https://github.com/Yenthe666/pysolaredge/blob/c8fa314ed59109c0b3032f88e5ccdcd16da41cc3/solaredge/api/sites.py#L134-L149 
I guess this:
```
   api_endpoint = '/site/%s/energy' % site_id
```

Should become:
```
   api_endpoint = '/site/%s/energyDetails' % site_id
```

Right?

**Comment by chrisjohnson00** [comment]
Yes.  I'll PR that.  I've also added 2 other endpoints that I'll include.
So to confirm, you are okay with breaking the `get_energy_details` method?  It will respond with a different payload structure after swinging it to the correct endpoint.

**Comment by Yenthe666** [comment]
Yeah I'm fine with that because it was wrong in the first place anyways. Looking forward to your PR :)

**Comment by chrisjohnson00** [comment]
resolved by pr #2
